tests/SprykerEcoTest/Zed/PunchoutCatalogs/_support/PunchoutCatalogsBusinessTester.php

```diff
@@ -86,4 +101,82 @@ public function retrieveConnectionPasswordFromVault(int $idPunchoutCatalogConnec
                 $idPunchoutCatalogConnection
             );
     }
+
+    /**
+     * @param int $idCompanyBusinessUnit
+     *
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionSetupTransfer
+     */
+    public function createPunchoutCatalogsConnectionSetupTransfer(int $idCompanyBusinessUnit): PunchoutCatalogConnectionSetupTransfer
+    {
+        return (new PunchoutCatalogConnectionSetupTransfer())
+            ->setLoginMode(static::CONNECTION_SETUP_LOGIN_MODE)
+            ->setFkCompanyBusinessUnit($idCompanyBusinessUnit);
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionCartTransfer
+     */
+    public function createPunchoutCatalogsConnectionCartTransfer(): PunchoutCatalogConnectionCartTransfer
+    {
+        return (new PunchoutCatalogConnectionCartTransfer())
+            ->setDefaultSupplierId(static::CONNECTION_CART_SUPPLIED_ID)
+            ->setMaxDescriptionLength(static::CONNECTION_CART_MAX_DESCRIPTION_LENGTH)
+            ->setMapping(static::CONNECTION_CART_MAPPING)
+            ->setEncoding(static::CONNECTION_CART_ENCODING);
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\CompanyBusinessUnitTransfer $companyBusinessUnitTransfer
+     *
+     * @return \Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer
+     */
+    public function createPunchoutCatalogConnection(CompanyBusinessUnitTransfer $companyBusinessUnitTransfer): PunchoutCatalogConnectionTransfer
```

Comment:
`createPunchoutCatalogConnectionTransfer`?

Reply:
I do not think it's the good name for it because it creates `PunchoutCatalogConnection ` in persistence, but not just a transfer.